### DISCUSSION TITLE
How to capture editor markdown shortcut event?

### DISCUSSION THREAD

**Discussion by GeekyADAMS** [ueberdosis/tiptap#2828] [General Questions]
I want to be able to capture markdown shortcut events when the user applies them, but I can't seem to find a way to do that.

**Comment by rfgamaral** [comment]
I don't think you can "capture" those events.

The Markdown shortcuts are based on input/paste rules (look through the source code to understand what they are), and the only way I can think of for you to "capture" them, is for you to override every extension with those rules, and insert a callback in here, but even that I'm not sure if it's possible.

What exactly are you trying to do? Maybe there's a better way to achieve your goal.
